fake/derived/path/Logs/Test/LogStoreManifest.plist

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>logFormatVersion</key>
+	<integer>10</integer>
+	<key>logs</key>
+	<dict/>
+</dict>
+</plist>
```

Comment:
Same here 🤷‍♂️ I should look into where these are coming from and why they weren't cleaned up

Reply:
Same happens here: these are also not cleaned up when I run tests 😅

Reply:
Thanks, Josh! 🤗 https://github.com/fastlane/fastlane/pull/17559